Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/jee/jsf/UserAuthorizationListener.java

```diff
@@ -141,9 +139,9 @@ public PhaseId getPhaseId() {
 
     private String formatPageName(String pageName) {
         if (pageName.startsWith("/")) {
-            return pageName.substring(1, pageName.length()).toLowerCase();
+            return pageName.substring(1, pageName.length()).toLowerCase(Locale.ENGLISH);
```

Comment:
Fortify only complain about comparing. We should stick to default toLowerCase and don't use "local.english".  We only need to check comparing by using equalIgnoreCase without using local.english. Check checkConfiguredDisplay() method and always to use "constant".equalIgnoreCase("stringtocompare")